apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/ErrorReporting/ErrorReportingExtension.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright 2017 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using Google.Api.Gax;
+using Google.Cloud.ErrorReporting.V1Beta1;
+using Google.Protobuf;
+using Google.Protobuf.WellKnownTypes;
+
+namespace Google.Cloud.Diagnostics.Common
+{
+    /// <summary>
+    /// Extensions for Error Reporting.
+    /// </summary>
+    internal static class ErrorReportingExtension
+    {
+        /// <summary>
+        /// Convert a <see cref="ReportedErrorEvent"/> to a <see cref="Struct"/>.
+        /// </summary>
+        public static Struct ToStruct(this ReportedErrorEvent errorEvent)
+        {
+            GaxPreconditions.CheckNotNull(errorEvent, nameof(errorEvent));
+            var jsonFormatter = new JsonFormatter(JsonFormatter.Settings.Default);
```

Comment:
You shouldn't need to construct the JsonFormatter explicitly here. Just JsonFormatter.Default should be fine.
It's possible that I should have a ToStruct method as an extension on IMessage at some point to avoid actually formatting and parsing - this is certainly going to be inefficient.